scripts/heat/ping_template.sh

```diff
@@ -43,3 +43,18 @@ cat >> /usr/bin/ping_forever_neighbour << EOF
 EOF
 chmod +x /usr/bin/ping_forever_neighbour
 /usr/bin/ping_forever_neighbour &
+
+cat >> /usr/bin/cinder_test << EOF
+  #!/bin/sh
+
+  sudo mkfs -t ext4 -L cinder_volume /dev/vdb
```

Comment:
This is just me not being used to work with volumes, but how are we sure that it's always `/dev/vdb`?